server/controllers/notification.js

```diff
@@ -0,0 +1,82 @@
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /notifications/
+// @desc create a new notification
+// @access Public
+
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  const { type, title, description } = req.body;
```

Comment:
Optional: As a bonus, it would nice to validate the parameters from the request body/route and return a 400 if they're missing

Reply:
I added them and tested in postman with empty params, they are sending back 400.